{"uuid":"87db80be-5c9c-49aa-8823-eb22bbbdb591","historyId":"4513b23d526162fbf83c18e1dea14d77","fullName":"Array: The user is able to submit code in tryEditor for Find Numbers with Even Number of Digits","labels":[{"name":"tag","value":"arrays"},{"name":"host","value":"LAPTOP-RSES787T"},{"name":"thread","value":"33172@LAPTOP-RSES787T.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is able to submit code in tryEditor for Find Numbers with Even Number of Digits"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is able to submit code in tryEditor for Find Numbers with Even Number of Digits"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature"},{"name":"package","value":"src.test.resources.features.5arrays_feature.Array"}],"links":[],"name":"The user is able to submit code in tryEditor for Find Numbers with Even Number of Digits","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Result do not match expected [submission success] but found [Error occurred during submission]","trace":"java.lang.AssertionError: Result do not match expected [submission success] but found [Error occurred during submission]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:160)\r\n\tat ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:150)\r\n"},"stage":"finished","steps":[{"name":"Given  The user is on \"Question\" page of \"Find Numbers with Even Number of Digits\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673283488931,"stop":1673283489220},{"name":"When  The user enter valid python code in tryEditor from sheet \"pythonCode\" and 7 for the question","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673283489220,"stop":1673283491571},{"name":"And  The user clicks on Submit button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673283491571,"stop":1673283491669},{"name":"Then  The user should be presented with successful submission message","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Result do not match expected [submission success] but found [Error occurred during submission]","trace":"java.lang.AssertionError: Result do not match expected [submission success] but found [Error occurred during submission]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:160)\r\n\tat ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:150)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673283491669,"stop":1673283492308}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"7"}],"start":1673283488927,"stop":1673283492677}